src/components/data/LocalVariablesContent.js

```diff
@@ -18,13 +18,31 @@ import * as StateHistory from 'src/libs/state-history'
 import * as Utils from 'src/libs/utils'
 
 
+export const getDisplayedAttribute = arr => {
+  const { key, value, description } = _.assign(...arr, {})
+  return [key, value, description ?? '']
+}
```

Comment:
Consider:
```suggestion
export const getDisplayedAttribute = arr => {
  const { key, value, description = '' } = _.mergeAll(arr)
  return [key, value, description]
}
```

Reply:
Oh nice! I had hoped there'd be something like `mergeAll` but it doesn't seem to show up on the [page I use as reference](https://lodash.com/docs/4.17.15), probably because it doesn't list the newest version's. Where do you look for them?

Reply:
It's actually an [fp-specific function](https://github.com/lodash/lodash/wiki/FP-Guide), created to handle the fact that non-fp-merge can take any number of arguments, but all fp functions are fixed arity.

Reply:
(unfortunately, it doesn't show up on the [generated fp docs](https://gist.github.com/jfmengels/6b973b69c491375117dc) for that reason, as well)

Reply:
> It's actually an fp-specific function, created to handle the fact that non-fp-merge can take any number of arguments, but all fp functions are fixed arity.

aaahhh…that explains the weirdness I was observing when playing with the FP version and comparing the result to the examples from the non-FP version 🤦🏻‍♂️

Reply:
Made the change - thanks!